CC/Sounder/files/conf-bs-and-two-client.json

```diff
@@ -0,0 +1,51 @@
+{
+        "BaseStations" : {
+                 "cells" : 1,
+                 "sdr_id" : ["files/iris-serials.txt"],
+                 "hub_id" : "files/hub-serials.txt",
+		 "frequency" : 2.5e9,
+                 "channel" : "AB",
+                 "rxgainA" : 65,
+                 "txgainA" : 87,
+                 "rxgainB" : 65,
+                 "txgainB" : 87,
+                 "rate" : 5e6,
+                 "frame_schedule" : [
+                    "BGGGGGPPPPGGGGGGGGGG"
+                 ],
+                 "max_frame" : 4000,
+                 "ofdm_symbol_per_subframe" : 10,
+                 "fft_size" : 64,
+                 "cp_size" : 16,
+                 "prefix" : 160,
+                 "postfix" : 160,
+                 "beamsweep" : true,
+                 "beacon_antenna" : 0
+        },
+
+        "Clients" : {
+                 "sdr_id" : [
+                     "RF3E000030",
+                     "RF3E0000xx"
```

Comment:
Is this intentional?

Reply:
unfortunately yes. we don't have another board in POWDER that we can use with this config file (since the EBC node is independent) so I just prefer to have it throw an error. I can change it if you have any suggestions though